kratos/includes/mortar_classes.h

```diff
@@ -1315,12 +1330,12 @@ class DualLagrangeMultiplierOperators
     }
     
     /**
-     * Calculates the matrix Ae
+     * Calculates the matrix Ae. To avoid problems in the inversion the matrix is normalized
```

Comment:
please refer in the comment to which article you are referring to and which formula (same for CalculateDe or whatever it was called)

Reply:
I am refering the whole thesis, i can say the apge and equation